CONTRIBUTING.md

```diff
@@ -32,3 +32,38 @@ Where `<library>` is the name of the library you want to build the docs for. For
  ```bash
 make docs LIB=platform/alz
 ```
+
+# Release process
+
+Once the PR has been merged, a new release must be created that follows the Library and CalVer versioning scheme.
+
+This process differs slightly depending on which library is being released.
+
+## ALZ Library
+
+1. Create a release with a tag name that follows the format `platform/alz/yyyy.mm.patch`, where `yyyy` is the year, `mm` is the month, and `patch` is the patch number (e.g., `platform/alz/2025.09.2`).
+    The year and month should only be updated when there is a policy refresh or major breaking change.
+2. The release title should be the same as the tag name.
+3. The release description should include a summary of the changes made in the release.
+4. For breaking changes, these must be called out in the release notes.
+5. Publish the release.
```

Comment:
```suggestion
5. Publish the release, select the *latest* option.
```

Reply:
Should ALZ always be the latest and AMBA / SLZ not have that set? It is the default, so would need to explicitly uncheck it for the others?